rad/monadic/registry.rad

```diff
@@ -0,0 +1,125 @@
+;; Helpers for setting up and interacting with the registry machine (as defined in
+;; rad/monadic/registry-remote.rad)
+{:module  'monadic/registry
+ :doc     "Functions for interacting with registry machine"
+ :exports '[create-registry-machine!]}
+
+(import prelude/lens :unqualified)
+(import prelude/machine '[send-code! send-signed-command!] :unqualified)
+(import prelude/time '[install-fake-clock] :unqualified)
+(import prelude/io :as 'io)
+(import prelude/key-management '[use-fake-keys! get-keys!] :unqualified)
+
+(def radicle-registry-machine-id
+  "The name of this machine."
+  "monadic/radicle/registry")
+
+(def create-registry-machine!
+  "Create a remote registry machine with the given url."
+  (fn []
+    (def owner-keys (lookup :public-key (get-keys!)))
+    (def id (machine/new-machine!))
+    (machine/send-prelude! id)
+    (send-code! id (find-module-file! "monadic/registry-remote.rad"))
+    (machine/send-signed-command! id radicle-registry-machine-id 'add-admin {:key owner-keys})
+    id))
+
+(def simple-project
+  "Register an open project with no labels."
+  (fn [i]
+    (def t (now!))
+    (<> {:labels      []
+         :state       :open
```

Comment:
Again, I think not having `open`/`closed` as state, but just removing from the ref, makes more sense.